src/app/manage-users/manage-user-updation/manage-user-updation.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { User } from 'src/app/shared/users.model';
+import { UsersService } from 'src/app/shared/users.service';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-manage-user-updation',
+  templateUrl: './manage-user-updation.component.html',
+  styleUrls: ['./manage-user-updation.component.css']
+})
+export class ManageUserUpdationComponent implements OnInit {
+
+  currentUser: Observable<User>;
+  userId: string;
+  
+  constructor(private usersService: UsersService, private route: ActivatedRoute) { }
+
+  ngOnInit(): void {
+
+    this.route.params.subscribe(
+      (param) => {
+        this.userId = param['id'];
+        this.currentUser=this.usersService.getUserDetails(param['id']);
```

Comment:
You already have a variable `this.userId` which holds the id from the route. So `params['id']` can be replaced with `this.userId`

Reply:
done this